inference-engine/thirdparty/clDNN/src/include/reorder_inst.h

```diff
@@ -33,11 +35,19 @@ struct typed_program_node<reorder> : public typed_program_node_base<reorder> {
     void requires_reinterpret(bool val) { req_reinterpr = (optimized && val); }
 
     void set_input_offset(tensor const& io) { input_offset = io; }
+    void record_input_layout(layout const& lo) { input_layout = lo; }
```

Comment:
sorry it was me that named this first.. , but how about rename this to "set_input_layout" so that the naming convention is kept with others?

Reply:
Ok, I will modify it.